docs/migration/4_0.md

```diff
@@ -1,3 +1,22 @@
 # Technical Changes in Spartacus 4.0
 
+### OccConfigLoaderModule
+- `OccConfigLoaderModule` was removed.
+
+### OccConfigLoaderService
+- `OccConfigLoaderService` was removed.
+
+### OccLoadedConfigConverter
+- `OccLoadedConfigConverter` was removed.
+
+### OccLoadedConfig 
+- `OccLoadedConfig` was removed.
+
+### OccSitesConfigLoader 
+- `OccSitesConfigLoader`was removed.
+
+### OccEndpoints
+`baseSitesForConfig` property has been deprecated and no longer exists.
```

Comment:
For this documentation to be helpful for customers, let's mention the replacement `Yyyy was removed. Please use xxxx instead`.

*The same holds for the code comments injected by schematics in customer's codebase.*

---
You are able to find the replacement info in the JSDoc of the deprecated classes, for example:
```
/**
 * @deprecated since 3.2 - use `SiteContextConfigInitializer` instead
 */
.............
export class OccConfigLoaderService {
````

Reply:
Can we also enchance the comments injected by the schematics?